### PR TITLE
The beginning - Converts meatpackers gateway mission to ruin

### PR DESCRIPTION
**Pull request by AffectedArc07**
## What Does This PR Do
Converts the blackmarket meat packers gateway mission to a space ruin.

## Why It's Good For The Game
This is a better way to handle killing the gateway, slowly move stuff away so each move can be refined.

I did this one first because its small (60x49) tiles, and already close enough to being a space ruin given the map surrounding. **The ruin content has not been modified apart from the removal of the gateway tiles, as well as area swaps**

## Images of changes
See MDB or MDB2

## Changelog
:cl: AffectedArc07
tweak: The gateway meat packers mission has been converted to a space ruin
/:cl:

### CODE REVIEW

File: paradise.dme [hunk 1]
```diff
@@ -1,2589 +1,2589 @@
...
-#include "code\modules\world_topic\manifest.dm"
-#include "code\modules\world_topic\ping.dm"
-#include "code\modules\world_topic\players.dm"
-#include "code\modules\world_topic\queue_status.dm"
-#include "code\modules\world_topic\status.dm"
-#include "goon\code\datums\browserOutput.dm"
-#include "interface\interface.dm"
-#include "interface\skin.dmf"
-// END_INCLUDE
+// DM Environment file for paradise.dme.
```
Comment (AffectedArc07) [inline paradise.dme:1]: I hate line endings

**Review by S34NW — CHANGES_REQUESTED** [review]
You need to use `/area/template_noop` not `/area/space` for empty turfs around the ruin please.

**Comment by AffectedArc07** [comment]
> You need to use `/area/template_noop` not `/area/space` for empty turfs around the ruin please.

Oh theres an area too?

**Review by S34NW — COMMENTED** [review]
image
Please also bump APCs, I'm not really sure how they are not bumped in the map file but are in-game currently, but please make the correction if possible :)

image
This door is also wonky, and a wire leads to nowhere

image
There is an awaystart landmark here that can go away,

image
this sink

**Comment by AffectedArc07** [comment]
> Guessing that little asteroid in this mission north of the ship that has the dead doctor and their full medical access ID is removed with this?

That


Doesnt exist on this gateway
image

File: code/game/area/areas/ruins/blackmarketpackers.dm [hunk 2]
```diff
@@ -1,26 +1,23 @@
 //Packer Ship Areas
 
-/area/awaymission/BMPship
+/area/ruin/unpowered/BMPship
 	name = "\improper BMP Asteroids"
 	icon_state = "away"
```
Comment (hal9000PR) [inline code/game/area/areas/ruins/blackmarketpackers.dm:5]: might want to change the icon state of these.

Comment (AffectedArc07) [reply code/game/area/areas/ruins/blackmarketpackers.dm:5]: Some of the ruins use the `away` icon and I plan to convert them all to just `ruin1` `ruin2` when we fully nuke the gateway. Can add those states now if you want.

Comment (hal9000PR) [reply code/game/area/areas/ruins/blackmarketpackers.dm:5]: preferably you would add them now.

Comment (AffectedArc07) [reply code/game/area/areas/ruins/blackmarketpackers.dm:5]: Will do

**Review by hal9000PR — APPROVED** [review]

File: code/game/area/areas/ruins/blackmarketpackers.dm (hunk 2 above)
Comment (AffectedArc07) [reply code/game/area/areas/ruins/blackmarketpackers.dm:5]: image

**Pull request merged**